README.md

```diff
@@ -196,15 +197,26 @@ cp /etc/armadillo/application.yml armadillo2-backup/application-armadillo2.yml
 N.B.change /usr/share to path matching your local config.
 
 ##### 3. Export data from Armadillo 2
-This step will copy Armadillo 2 data from minio into the folder matching of an Armadillo 3 data folder:
+Look up the user/password in the application.yml of the old armadillo. They're called minio access key and  minio
+secret key.
+```
+cat /root/armadillo2-backup/application-armadillo2.yml
+```
 
+Do the following step in a separate screen (use the `screen` command to create it and press enter or space when a 
+message shows up).
```

Comment:
Prepend maybe "The following step will take a long time depending on your data size. To make sure this does not fail we run it in a detached screen"

Is screen always installed on Ubuntu?
Why not run it using `nohup`? Guess because a screen you can read back :)